src/Middleware/RequestAuthorizationMiddleware.php

```diff
@@ -24,6 +24,7 @@
 use Psr\Http\Server\MiddlewareInterface;
 use Psr\Http\Server\RequestHandlerInterface;
 use RuntimeException;
+use Zend\Diactoros\Response\RedirectResponse;
```

Comment:
This should be `Laminas` now not `Zend`. Also it would require adding `laminas/diactoros` as a composer dependency.

Reply:
If any of those ship with it, we might not need the verbose addition in root composer json here: https://github.com/cakephp/authorization/blob/master/composer.json#L13-L17

Reply:
They don't.